### PR TITLE
Media Master tutorial 

### PR DESCRIPTION
**Pull request by devmrfitz**

### CODE REVIEW

File: media-master/blog.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
+# Media Master
+
+## Background
+
+Currently, to process any image or video, the server’s need to process them while they are being uploaded. This takes a lot of processing power and memory and eventually slows down the uploading process. I created this project to apply processing on the videos and images after they are uploaded to the temporary folder and then transfer the processed files to permanent storage. The project uses Courier and Azure Functions.
```
Comment (shreythecray) [inline media-master/blog.md]: Cool project! You need to add some more context in the beginning for the reader to understand why this is important. Why are we looking at processing images and videos? Why does it matter that it take time - who does that impact and why is that bad? Including examples or analogies would help.

I'd also recommend adding either screenshots/gifs or embedding your demo video at the top.

File: media-master/blog.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+
+Any website that accepts media content (photos, videos, etc) does some or the other form of background processing on said media. If this processing runs in the main thread of the web server, it could lead to horrible response times and also increase susceptibility to a Denial of Service attack. Operations like compression, validation, re-encoding and watermarking are often essential to services of many popular web apps. Even more important is the surety of being notified whenever something goes wrong in crucial operations like these.
+
+**<u>Tools used</u>** 
+
+- Azure Functions: Azure Functions is the ideal tool for asynchronous background tasks like ours.
```
Comment (shreythecray) [inline media-master/blog.md]: Need more context on this

File: media-master/blog.md [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+
+- Courier: Courier provides an amazing suite of notification integrations.
+
+### Roadmap
+
+- Storing the files
```
Comment (shreythecray) [inline media-master/blog.md]: Maybe number these?

File: media-master/blog.md [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+
+![Image description](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/ftdtrqxwkgfre9dhjs14.png)
+
+These variables can finally be fed into [Courier's Python SDK](https://pypi.org/project/trycourier/) to facilitate simple notification sending.
+
+Courier works by taking in an event as input via an API, which is in our case, a successful upload. The event is accompanied with the data required for the feedback and details of the recipient. It then generates a notification and sends it through the channel specified. Here, I have chosen emails to be the channel for receiving notification.
+
+Imported trycourier package from Courier service and then added all the metadata like receiver’s email, subject, body content, and the specific channel.  
+
+```
```
Comment (shreythecray) [inline media-master/blog.md:116]: Breakdown the code

File: media-master/blog.md [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+
+    logging.warning(f"Unidentified video: {e}")        
+
+     
+
+```
+
+## Conclusion
+
+So, our fast, light-weight and easy-to-use media processing service is ready to used and can help a lot of students and professionals in their day-to-day hustle.
```
Comment (shreythecray) [inline media-master/blog.md]: Love it!

**Review by shreythecray — CHANGES_REQUESTED** [review]
Cool project!

File: media-master/blog.md (hunk 1 above)
Comment (devmrfitz) [reply media-master/blog.md]: Done

File: media-master/blog.md (hunk 5 above)
Comment (devmrfitz) [reply media-master/blog.md]: Thanks a lot!

File: media-master/blog.md (hunk 4 above)
Comment (devmrfitz) [reply media-master/blog.md:116]: Done

**Comment by devmrfitz** [comment]
@shreythecray Resolved all suggestions. Does it look good now?

**Review by shreythecray — APPROVED** [review]

**Pull request merged**